WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductSelectorView.swift

```diff
@@ -125,6 +125,9 @@ struct ProductSelectorView: View {
                 if let cancelButtonTitle = configuration.cancelButtonTitle {
                     Button(cancelButtonTitle) {
                         isPresented.toggle()
+                        if !isPresented {
+                            viewModel.clearSelection()
+                        }
```

Comment:
Super nit: 

If the purpose of this button is to set `isPresented` as false. For slightly better readability, can we rewrite this into 
```suggestion
                        isPresented = false
                        viewModel.clearSelection()
```

Reply:
In this case we're not setting `isPresented` but just checking if it is false after being toggled. I guess I can use a ternary operator here as well for readability 👍